corehq/util/log.py

```diff
@@ -287,3 +289,8 @@ def draw(position):
     end = datetime.now()
     print "Finished at {:%Y-%m-%d %H:%M:%S}".format(end)
     print "Elapsed time: {}".format(display_seconds((end - start).total_seconds()))
+
+
+def send_HTML_email(subject, recipient, html_content, *args, **kwargs):
+    kwargs['ga_track'] = kwargs.get('ga_track', False) and analytics_enabled_for_email(recipient)
```

Comment:
That's a tough one - I'm not sure how to handle that.  I don't think we can include the tracking link for some users and not others.  We could skip the tracking if any of the cc'd users have opted out.  @dimagi/product @krsdimagi What do you think?